src/explorer/SiteTreeItem.ts

```diff
@@ -138,18 +138,23 @@ export abstract class SiteTreeItem extends SiteTreeItemBase implements ISiteTree
         await deleteSite(this.root.client);
     }
 
-    public async isHttpLogsEnabled(): Promise<boolean> {
+    public async isLogsEnabled(): Promise<boolean> {
         const logsConfig: WebSiteManagementModels.SiteLogsConfig = await this.root.client.getLogsConfig();
-        return !!(logsConfig.httpLogs && logsConfig.httpLogs.fileSystem && logsConfig.httpLogs.fileSystem.enabled);
+        return !!(logsConfig.httpLogs?.fileSystem?.enabled && logsConfig.applicationLogs?.fileSystem?.level === 'Verbose');
```

Comment:
I don't think we should be checking that it's "Verbose" here. As long as logging enabled, we should stream logs as-is and users can do whatever log level they want